.travis.yml

```diff
@@ -35,12 +35,11 @@ before_install:
         brew unlink python@2 &&
         brew install swig openblas bison flex cminpack ceres-solver dlib nlopt tbb &&
         export HOMEBREW_PREFIX=`brew --prefix`;
+        pip3 install matplotlib scipy
     fi
 
 install:
-# use system python
-  - export PATH=/usr/bin:$PATH
```

Comment:
I believe it was removing this export PATH=/usr/bin:$PATH that unexpectedly make the cblas detection fail